operations/finance/spending-company-money/staff-member-expenses/internship-expense-policy.md

```diff
@@ -0,0 +1,27 @@
+# Internship Expense Policy
+
+At Mattermost, we want our interns to be successful from Day 1 and you'll be provisioned an Expensify account. As a general rule, our [expense policy](https://handbook.mattermost.com/operations/finance/spending-company-money/expense-policy) for our full-time and part-time staff members is the same for our interns with some modifications. As an intern, you can refer to our How to spend company money policy for guidance for most expense-related questions.
+
+The purpose of this section is to identify the variance from our full time staff member spending policy.
+
+## Notable spending differences for interns
+
+In general, we are a culture where you don’t have to ask permission before making purchases on behalf of the company unless there are guidelines specific on what and how much can be purchased. If in doubt, ask your manager before you make any purchases to confirm you are following the purchasing guidelines. A good reference point for spending is the [expense policy](https://handbook.mattermost.com/operations/finance/spending-company-money/expense-policy) section if you are uncertain.
```

Comment:
As per above: The "expense policy" URL isn't working. Is it still to be added? Or will it be this one: #501?

Reply:
Please see above comment